.github/workflows/workspace-integration-tests.yml

```diff
@@ -0,0 +1,179 @@
+name: "Workspace integration tests"
+on:
+    workflow_dispatch:
+        inputs:
+            name:
+                required: true
+                description: "The name of the preview environment"
+            version:
+                required: true
+                description: "The version of Gitpod to install"
+            skip_deploy:
+                required: false
```

Comment:
can add `type: boolean` so that it shows as a toggle instead of a text field in the github UI